handlers/ticket.go

```diff
@@ -726,6 +728,8 @@ func (th *ticketHandler) ProcessTicketReview(w http.ResponseWriter, r *http.Requ
 		Description: reviewReq.Value.TicketDescription,
 		Status:      existingTicket.Status,
 		Version:     existingTicket.Version + 1,
+		Amount:      reviewReq.Value.TicketAmount,
```

Comment:
Again just want to check the logic here. the Stakwork workflow doesn't update the amount or category so I don't want to update based on the post request from stakwork but instead just retain the value that was already in the ticket. 

We do this with other fields already that aren't udpated by the workflow. 

E.g. `Sequence:    existingTicket.Sequence,`